### ISSUE TITLE
postgresql-dev unsatisfiable constraints 

### ISSUE THREAD

**Issue by join3r** [ninech/netbox-docker#72]
docker-compose up fails during build of netbox image.

```
ERROR: unsatisfiable constraints:
  libressl-dev-2.6.4-r1:
    conflicts:
               openssl-dev-1.0.2o-r0[pc:libcrypto=2.6.4]
               openssl-dev-1.0.2o-r0[pc:libssl=2.6.4]
               openssl-dev-1.0.2o-r0[pc:openssl=2.6.4]
    satisfies:
               postgresql-dev-10.4-r0[libressl-dev]
               openldap-dev-2.4.45-r3[libressl-dev]
  openssl-dev-1.0.2o-r0:
    conflicts:
               libressl-dev-2.6.4-r1[pc:libcrypto=1.0.2o]
               libressl-dev-2.6.4-r1[pc:libssl=1.0.2o]
               libressl-dev-2.6.4-r1[pc:openssl=1.0.2o]
    satisfies: world[openssl-dev]
```

Issue is inherited from alpine base image which is used by python:3.6-alpine

**Comment by tobiasge** [comment]
This will be fixed when pull request #70 is merged.

**Comment by calmyournerves** [comment]
Fixed!
